### PR TITLE
Verisure

### PR DESCRIPTION
**Pull request by practiz**

### CODE REVIEW

**Comment by wborn** [comment]
@practiz Why does this PR also touch Tellstick Binding files?

File: addons/binding/pom.xml [hunk 1]
```diff
@@ -58,6 +58,8 @@
     <module>org.openhab.binding.toon</module>
     <module>org.openhab.binding.urtsi</module>
     <module>org.openhab.binding.vitotronic</module>
+    <!--module>org.openhab.binding.zwave</module-->
```
Comment (wborn) [inline addons/binding/pom.xml]: Zwave Binding users will not approve this change :-) You do need to add your own binding as a module here.

**Comment by wborn** [comment]
When this is squashed and signed off please also [add an Also-by:](https://wiki.eclipse.org/Development_Resources/Contributing_via_Git#The_Commit_Record) for @jarlebh.

Here's a link to the [community thread](https://community.openhab.org/t/testers-for-verisure-openhab-2-binding) for people who are interested/testing this Binding. :+1:

**Comment by jarlebh** [comment]
I pushed my latest changes (which is squashed).

**Comment by martinvw** [comment]
Who is the creator / owner of this PR?

**Comment by wborn** [comment]
Can you also remove the [.DS_Store](https://en.wikipedia.org/wiki/.DS_Store) files @jarlebh? I've also seen a lot of generated JavaDocs that do not have any added value.

**Comment by jarlebh** [comment]
It is my code, i guess practiz thought it should be merged. It is almost ready

**Comment by martinvw** [comment]
Please include the binding in the parent `pom.xml` in that case it will already be tested and analysed by travis. And it will become available in [jfrog builds list](https://openhab.jfrog.io/openhab/libs-pullrequest-local/org/openhab/binding/).

Please also check:

http://docs.openhab.org/developers/development/bindings.html
http://docs.openhab.org/developers/development/guidelines.html

And it we would be great if you could also review the report of [the Static code analysis](https://docs.openhab.org/developers/development/bindings.html#static-code-analysis), this also saves me some time while reviewing.

**Comment by jarlebh** [comment]
There is one warning that I don't understand:
[WARNING] .binding.verisure\META-INF\MANIFEST.MF:[0]
The package org.openhab.binding.verisure.discovery should be marked as "internal" if it is not exported.

**Comment by wborn** [comment]
> [WARNING] .binding.verisure\META-INF\MANIFEST.MF:[0]
The package org.openhab.binding.verisure.discovery should be marked as "internal" if it is not exported.

It's an OSGi convention so developers can easily see from the package name if classes in the package are exposed to other bundles @jarlebh .

So if you just move the discovery package inside the internal package it will be solved. 

Just look at the [Harmony Binding](https://github.com/openhab/openhab2-addons/tree/master/addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/internal/discovery) for instance where the Discovery classes are also inside the internal package.

**Comment by jarlebh** [comment]
The build error does not seems to related to my code

**Comment by martinvw** [comment]
Reopening to trigger rebuild

**Comment by tnemrap** [comment]
Any updates on this?

/mike

**Comment by martinvw** [comment]
@jarlebh can you open a pr on your own behalf for this PR?

**Pull request closed without merging**